tools/mercury/README.md

```diff
@@ -0,0 +1,520 @@
+# Mercury
+##interactive Golang Testing System for Qpid Dispatch Router
+
+
+
+<br/>
+
+
+###Requirements
+
+1. the go language on your machine
+2. installed proton and dispatch router software
+
+
+
+<br/>
+
+###Mercury Audience
+
+I see Mercury as a tool for developers. The user has one or more installed versions of the Dispatch Router + Proton code and wantsto easily set up a complex network including Dispatch routers, edge routers, and clients with nontrivial addressing patterns. Currently I think that creation of nontrivial networks is difficult enough that it discourages extensive testing during development.
```

Comment:
Can we remove "I see". Can it simply be "Mercury is a tool...."

Also remove "I think" etc.

Reply:
Well ... I was planning to make all those occurrences of the word "I" into hyperlinks that would show my picture, contact information, LinkedIn page, a small biography, and an audio file of me singing _None Shall Sleep_ from the final act of the opera _Turandot_.   

But, OK.   I took them out.